include/deal.II/cgal/utilities.h

```diff
@@ -198,6 +203,37 @@ namespace CGALWrappers
     const CGAL::Surface_mesh<CGALPointType> &surface_mesh_2,
     const BooleanOperation &                 boolean_operation,
     CGAL::Surface_mesh<CGALPointType> &      output_surface_mesh);
+
+  /**
+   * Remesh a CGAL::Surface_mesh according to a set of given named parameters.
+   * See more about them at:
+   * https://doc.cgal.org/latest/Mesh_3/classCGAL_1_1Mesh__criteria__3.html.
+   *
```

Comment:
Can you copy here the blob about the parameters that we added to the other functions? We don't want users to go elsewhere for the documentation if it is possible to have it right here...